openid/src/main/java/fish/payara/security/openid/controller/ConfigurationController.java

```diff
@@ -494,4 +488,19 @@ static CacheKey keyFromDefinition(OpenIdAuthenticationDefinition definition) {
         } : new String[5];
     }
 
+    private String getEnvironmentProperty(String property) {
```

Comment:
If I put in clientSecret of `b78d6xPATH`, should I get that replaced with value of PATH environment variable? I suppose not.